test/elasticsearch/indexing/bulk_test.exs

```diff
@@ -81,7 +81,7 @@ defmodule Elasticsearch.Index.BulkTest do
     end
   end
 
-  describe ".encode!/3" do
+  describe ".encode!/4" do
```

Comment:
Can you add a test that calls the function with the new `action_type` option set to "update" or something else?

Reply:
Sure thing, pushed! Let me know if those tests need to be changed or if you have any other suggestions.